h/static/styles/annotation.scss

```diff
@@ -10,6 +10,10 @@
   background-color: $white;
 }
 
+.annotation-quote-list--orphan {
```

Comment:
As per the [naming conventions](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#naming) documented in the new CSS style guide, the `--modifier` flags should only be used for static variations of a component. For changing the style of an element depending on some dynamic state, use an `is-$state` class. ie. `.annotation-quite-list.is-orphan`.

The reason for this is that it is obvious which classes are referenced in code and which are not.